community/kernel/src/main/java/org/neo4j/kernel/impl/store/StoreFactory.java

```diff
@@ -119,9 +119,9 @@ public void setPageCache( PageCache pageCache )
         this.pageCache = pageCache;
     }
 
-    public NeoStores openNeoStores( boolean createIfNotExist )
+    public NeoStores openNeoStoresLazily( boolean createIfNotExist )
     {
-        return openNeoStores( createIfNotExist, true );
+        return openNeoStores( createIfNotExist, false );
     }
 
     public NeoStores openNeoStores( boolean createIfNotExist, boolean eagerlyInitializeStores )
```

Comment:
I am wondering at this point if it isn't better having flags rather than booleans and make explicit when you allow creation/laziness in each call.